wpt

```diff
@@ -1,34 +1,11 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 if __name__ == "__main__":
     import sys
-    from tools.wpt import wpt
-    args, extra = wpt.parse_args(sys.argv[1:])
-    commands = wpt.load_commands()
-    py3only = commands[args.command]["py3only"]
+    if (sys.version_info.major < 3 or
+            (sys.version_info.major == 3 and sys.version_info.minor < 6)):
```

Comment:
FWIW I would prefer this to be less indented.

Reply:
As in, 4 spaces? That will make it aligned with the code block... if that's what you want then I'm happy to make that change, but it reads weird to me:

```
    if (sys.version_info.major < 3 or
        (sys.version_info.major == 3 and sys.version_info.minor < 6)):
        sys.stderr.write("wpt requires Python 3.6 or higher\n")
        sys.exit(1)
```

Reply:
Yeah, so apparently I disagree with some python tooling on this; the fact that `if (` is exactly four characters is unfortunate, but having the second line randomly extra indented looks worse to me.

Anyway I don't care enough to block or anything.